frontstage/views/register.py

```diff
@@ -34,6 +35,8 @@ def validate_enrolment_code(enrolment_code):
 
     if result.status_code == 200 and json.loads(result.text)['active']:
         case_id = json.loads(result.text)['caseId']
+    elif result.status_code != 200:
```

Comment:
👍 
Would it be worth maybe having a list of error codes so that when we get one we could pass it through that list then link it to that page, and if not it would go to the default?